code/style.css

```
@@ -1,3 +1,133 @@
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  background: rgb(253, 146, 166);
+  font-family: 'Montserrat', sans-serif;
+  font-weight: bold;
+  color: rgb(27, 20, 100);
+}
+
+h1 {
+  text-transform: uppercase;
+  display: inline;
+  display: flex;
+  justify-content: center;
+  font-size: 30px;
+  margin-top: 10px;
+}
+
+h3 {
+  border-bottom: solid 1px black;
+  margin-bottom: 0.5em;
+  font-size: 18px;
+}
+
+.profile-info {
+  padding-left: 1em;
+  display: flex;
+  font-size: 20px;
+  width: 100vw;
+  align-items: center;
+  padding-left: 5vw;
+  margin-top: 5px;
+  margin-bottom: 10px;
+}
+
+.project-container {
+  display: grid; 
+  grid-template-columns: 90vw;
+  border-top: double 3px rgb(27, 20, 100);
+  padding-left: 5vw;
+  padding-right: 5vw;
+  padding-top: 1vh;
+  padding-bottom: 5vh;
```

To make it a little shorter, you've could have wrapped it up in one line.
I also attend to do a lot of unnecessary lines of code:-)
```suggestion
  padding: 1vh 5vw 5vh 5vw;
```